scalding-core/src/test/scala/com/twitter/scalding/typed/RequireOrderedSerializationTest.scala

```diff
@@ -18,24 +18,25 @@ package com.twitter.scalding
 import com.twitter.scalding.serialization.CascadingBinaryComparator
 import com.twitter.scalding.serialization.OrderedSerialization
 import com.twitter.scalding.serialization.StringOrderedSerialization
+import com.twitter.scalding.serialization.RequireOrderedSerializationMode
 
 import org.scalatest.{ Matchers, WordSpec }
 
-class NoOrderdSerJob(args: Args) extends Job(args) {
+class NoOrderdSerJob(args: Args, mode: String) extends Job(args) {
```

Comment:
mode is confusing here since it is almost always `scalding.Mode` named that. Can we rename?